internal/service/ce/cost_category.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 	"github.com/hashicorp/terraform-provider-aws/internal/conns"
 	"github.com/hashicorp/terraform-provider-aws/internal/flex"
+	tftags "github.com/hashicorp/terraform-provider-aws/internal/tags"
```

Comment:
I think you're missing the `CustomizeDiff`
```suggestion
	tftags "github.com/hashicorp/terraform-provider-aws/internal/tags"
	"github.com/hashicorp/terraform-provider-aws/internal/tfresource"
	"github.com/hashicorp/terraform-provider-aws/names"
)
func ResourceCostCategory() *schema.Resource {
	return &schema.Resource{
		CreateContext: resourceCostCategoryCreate,
		ReadContext:   resourceCostCategoryRead,
		UpdateContext: resourceCostCategoryUpdate,
		DeleteContext: resourceCostCategoryDelete,
		Importer: &schema.ResourceImporter{
			StateContext: schema.ImportStatePassthroughContext,
		},
                CustomizeDiff: customdiff.Sequence(verify.SetTagsDiff),
```